### PR TITLE
How-to and style guide updates

### PR DESCRIPTION
**Pull request by CathPag**

### CODE REVIEW

File: content/en/style-guide/_index.md [hunk 1]
```diff
@@ -112,6 +114,8 @@ If merged, your submission will be the official CNCF definition for that term (u
 
 **Don't plagiarize**. The same rules apply to Glossary as to any other serious publication. Don't copy and paste other people's work unless you are quoting and contributing it to them. If you like a particular section of a definition, paraphrase it in your own words.
 
+**Reference authoritative resources**. When possible, link to authoritative resources such as project docs. Note that we cannot link to content developerd by vendors.
```
Comment (iamNoah1) [inline content/en/style-guide/_index.md:117]:
```suggestion
**Reference authoritative resources**. When possible, link to authoritative resources such as project docs. Note that we cannot link to content developed by vendors. 
```

**Review by iamNoah1 — COMMENTED** [review]
lgtm, except for a little typo.

File: content/en/contribute/_index.md [hunk 2]
```diff
@@ -83,7 +83,7 @@ Copy the content…
 
 ![create new file](/images/how-to/howto-09.png)
 
-Add the term name in the URL (no capitalization and no space) and .md at the end (note: if your preview doesn't work, you probably forgot to add .md at the end). Now paste the template content below. Copy and paste your definition into the file. Note that GitHub uses markdown to format the text (e.g., hyperlink, bold, italic). Please refer to this [markdown cheat sheet](https://www.markdownguide.org/cheat-sheet/). To verify you've used markdown as intended, go to "preview."
+Add the term name in the URL (no capitalization and no space) and .md at the end (note: if your preview doesn't work, you probably forgot to add .md at the end). Now paste the template content below. Copy and paste your definition into the file. To make reviews easier, please **use semantic line breaks** (one line per sentence). Note that GitHub uses markdown to format the text (e.g., hyperlink, bold, italic). Please refer to this [markdown cheat sheet](https://www.markdownguide.org/cheat-sheet/). To verify you've used markdown as intended, go to "preview."
```
Comment (jihoon-seo) [inline content/en/contribute/_index.md:86]: IMO "Semantic Line Breaks" is not necessarily equal to "one line per sentence".
So.. how about this? 😊

```suggestion
Add the term name in the URL (no capitalization and no space) and .md at the end (note: if your preview doesn't work, you probably forgot to add .md at the end). Now paste the template content below. Copy and paste your definition into the file. To make reviews easier, please **use [semantic line breaks](https://sembr.org/)** (e.g. one line per sentence). Note that GitHub uses markdown to format the text (e.g., hyperlink, bold, italic). Please refer to this [markdown cheat sheet](https://www.markdownguide.org/cheat-sheet/). To verify you've used markdown as intended, go to "preview."
```

**Review by jihoon-seo — COMMENTED** [review]
@CathPag PTAL!

**Review by iamNoah1 — APPROVED** [review]
lgtm

**Review by JasonMorgan — APPROVED** [review]

**Pull request merged**